lib/libxx/Makefile

```diff
@@ -59,19 +59,14 @@ CSRCS =
 
 CXXSRCS  = libxx_cxapurevirtual.cxx libxx_eabi_atexit.cxx  libxx_cxa_atexit.cxx
 
-ifeq (,$(findstring y,$(CONFIG_UCLIBCXX) $(CONFIG_LIBCXX)))
+ifneq ($(CONFIG_LIBCXX),y)
```

Comment:
neq?

Reply:
Yes. Those CXX sources are not required when LIBCXX is enabled.